raspberry/sensors/pressure_sensor.py

```diff
@@ -0,0 +1,182 @@
+# TODO: implement proper code for a particular sensor
+# TODO: add proper Doc strings
+# NOTE: for now, we are just passing dummy values
+
+from filters import median_filter
```

Comment:
From PEP8: Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports

Study [this](https://peps.python.org/pep-0008/#imports)